addons/my-ocular/addon.json

```diff
@@ -1,15 +1,41 @@
 {
-  "name": "Show my-ocular status",
-  "description": "Shows my-ocular statuses under usernames on the forums. Uses my-ocular.jeffalo.net to obtain status data. Statuses are moderated and filtered.",
+  "name": "Show ocular status",
+  "description": "Shows ocular statuses and favourite colours. Requests my-ocular.jeffalo.net to obtain status data. Statuses are moderated and filtered.",
```

Comment:
Favorite.

Reply:
Also, why did you change "uses" to "requests"?

Reply:
it flowed a big better, should I change it back?

Reply:
Yes, keep it as before

Reply:
done